python/bottle/webappWithTests/Tests/unit_tests/test_views.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+import sys
```

Comment:
have you removed the unittest.py file from inside the application folder and cleaned up all the unwanted test related files/dependencies?

Reply:
including unwanted dependencies in requirements.txt etc..